client/src/views/components/PayMethods/PayMethods.ts

```diff
@@ -99,15 +116,18 @@ export default class PayMethod {
   createCard(): string {
     return this.state
       .map((pay, idx) => {
+        let isInitialChoicedButton = '';
+        this.filter.card === pay.payMethodName ? (isInitialChoicedButton = 'active') : (isInitialChoicedButton = '');
```

Comment:
이렇게 리팩토링 할 수 있어 보입니다 :)

```suggestion
        isInitialChoicedButton = this.filter.card === pay.payMethodName ? 'active' : '';
```

Reply:
좋습니다!